FastMenu/index.tsx

```diff
@@ -18,9 +24,9 @@ const settings = definePluginSettings({
     },
 });
 
-const lazyLayers = [];
+const lazyLayers: any[] = [];
```

Comment:
This would be `string[]`, no?

Reply:
My fault I forgot to change it from testing.